src/Translator/src/DependencyInjection/Configuration.php

```diff
@@ -28,6 +28,14 @@ public function getConfigTreeBuilder(): TreeBuilder
         $rootNode
             ->children()
                 ->scalarNode('dump_directory')->defaultValue('%kernel.project_dir%/var/translations')->end()
+                ->arrayNode('excluded_domains')
+                    ->scalarPrototype()->end()
+                    ->defaultValue([])
+                ->end()
+                ->arrayNode('included_domains')
+                    ->scalarPrototype()->end()
+                    ->defaultValue([])
+                ->end()
```

Comment:
IMO if we configure the domains we want to exclude, we don't need to configure the domains we want to include, and vice-versa.

What about adding a validation telling the user that `excluded_domains` and `included_domains` can not be filled at the same time? :)